net.kuribo64.melonDS.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.kuribo64.melonDS.desktop</id>
+  <name>melonDS</name>
+  <summary>Nintendo DS and DSi emulator</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <content_rating type="oars-1.0" />
+  <description>
+    <p>
+		melonDS aims at providing fast and accurate Nintendo DS emulation. While it is still a work in progress, it has a pretty solid set of features.
+    </p>
+	<ul>
+		<li>Nearly complete core (CPU, video, audio, ...)</li>
+		<li>OpenGL renderer, 3D upscaling</li>
+		<li>RTC, microphone, lid close/open</li>
+		<li>Joystick support</li>
+		<li>Savestates</li>
+		<li>Various display position/sizing/rotations modes</li>
+		<li>Work-in-progress Wi-Fi emulation for online connectivity and local multiplayer</li>
+		<li>Experimental emulation of the Nintendo DSi</li>
+		<li>...and even more is planned!</li>
+	</ul>
+	<p>
+		<strong>Note:</strong> BIOS and firmware files dumped from a real DS or DSi console are required to use melonDS.
+	</p>
+  </description>
+  <url type="homepage">http://melonds.kuribo64.net/</url>
+  <screenshots>
+    <screenshot type="default">https://raw.githubusercontent.com/flathub/net.kuribo64.melonDS/master/screenshots/1.png</screenshot>
+    <screenshot type="default">https://raw.githubusercontent.com/flathub/net.kuribo64.melonDS/master/screenshots/2.png</screenshot>
+    <screenshot type="default">https://raw.githubusercontent.com/flathub/net.kuribo64.melonDS/master/screenshots/3.png</screenshot>
```

Comment:
This repo doesn't exists yet and so are the screenshots, the tests will fail for missing/not found screenshots.

Reply:
Should I change this for the time being so the tests will pass, or is this fine?

Reply:
Please change it so we can have the test build pass.

Reply:
Did that myself, I hope you don't mind.